File: Web/AFSport.Web.Core/Model/Caixa.cs
using System;
namespace AFSport.Web.Core.Model
{
    public class Caixa
    {
        public Caixa()
        {

        }

        public Caixa(decimal valorInicial, int idUsuario)
        {
            this.ValorInicial = valorInicial;
            this.IdUsuario = idUsuario;
            this.Data = DateTime.Now;
        }

        public int IdCaixa { get; set; }
        public decimal ValorInicial { get; private set; }
        public DateTime Data { get; set; }
        public int IdUsuario { get; set; }
        public Usuario Usuario { get; set; }

        public void SetValorInicial(decimal valor)
        {
            this.ValorInicial = valor > 0 ? valor : 0;
        }
    }
}

File: Mobile/AFSport/app/src/main/java/com/android/rafaelalves/afsport/activity/model/Caixa.java
package com.android.rafaelalves.afsport.activity.model;

import java.util.Date;

public class Caixa {
    private int idCaixa;
    private double valorInicial;
    private String dataString;
    private Usuario usuario;

    public int getIdCaixa() {
        return idCaixa;
    }

    public void setIdCaixa(int idCaixa) {
        this.idCaixa = idCaixa;
    }

    public double getValorInicial() {
        return valorInicial;
    }

    public void setValorInicial(double valorInicial) {
        this.valorInicial = valorInicial;
    }

    public String getDataString() {
        return dataString;
    }

    public void setDataString(String dataString) {
        this.dataString = dataString;
    }

    public Usuario getUsuario() {
        return usuario;
    }

    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }
}
